src/Veldrid/OpenGL/OpenGLCommandExecutor.cs

```diff
@@ -941,14 +941,20 @@ public void SetComputeResourceSet(uint slot, ResourceSet rs, uint dynamicOffsetC
                         glTexViewRW.EnsureResourcesCreated();
                         if (pipeline.GetTextureBindingInfo(slot, element, out OpenGLTextureBindingSlotInfo imageBindingInfo))
                         {
+                            var layered = texViewRW.Target.Usage.HasFlag(TextureUsage.Cubemap) || texViewRW.ArrayLayers > 1;
+
+                            //todo: validate and test binding subsets of cubemap arrays
+                            if (layered && texViewRW.BaseArrayLayer > 0)
```

Comment:
Should probably throw if `texViewRW.ArrayLayers < ArrayLayers` as well.  I'm not sure what might be intended if someone were to try to create a view of a subset of layers (e.g. 2 faces of a cubemap) but I don't think that's supported by OpenGL here.